integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/GenerateFiles.scala

```diff
@@ -0,0 +1,667 @@
+      var writer: DataFileWriter[GenericRecord] = null
+      var encoder: Encoder = null
+      var output: ByteArrayOutputStream = null
+      try {
+        val schema = new org.apache.avro.Schema.Parser().parse(avroSchema)
+        val reader = new GenericDatumReader[GenericRecord](schema)
+        input = new ByteArrayInputStream(json.getBytes())
+        output = new ByteArrayOutputStream()
+        val din = new DataInputStream(input)
+        writer = new DataFileWriter[GenericRecord](new GenericDatumWriter[GenericRecord]())
+        writer.create(schema, output)
+        val decoder = DecoderFactory.get().jsonDecoder(schema, din)
+        var datum: GenericRecord = reader.read(null, decoder)
+        return datum
+      } finally {
+        input.close()
+        writer.close()
+      }
+    }
+
+    /**
+     * this method returns true if local dictionary is created for all the blocklets or not
+     *
+     * @return
+     */
+    def getDimRawChunk(blockindex: Int,
+        storePath: String): util.ArrayList[DimensionRawColumnChunk] = {
+      val dataFiles = FileFactory.getCarbonFile(storePath)
+        .listFiles(new CarbonFileFilter() {
+          override def accept(file: CarbonFile): Boolean = {
+            if (file.getName
+              .endsWith(CarbonCommonConstants.FACT_FILE_EXT)) {
+              true
+            } else {
+              false
+            }
+          }
+        })
+      val dimensionRawColumnChunks = read(dataFiles(0).getAbsolutePath,
+        blockindex)
+      dimensionRawColumnChunks
+    }
+
+    def read(filePath: String, blockIndex: Int) = {
+      val carbonDataFiles = new File(filePath)
+      val dimensionRawColumnChunks = new
+          util.ArrayList[DimensionRawColumnChunk]
+      val offset = carbonDataFiles.length
+      val converter = new DataFileFooterConverterV3
+      val fileReader = FileFactory.getFileHolder(FileFactory.getFileType(filePath))
+      val actualOffset = fileReader.readLong(carbonDataFiles.getAbsolutePath, offset - 8)
+      val blockInfo = new TableBlockInfo(carbonDataFiles.getAbsolutePath,
+        actualOffset,
+        "0",
+        new Array[String](0),
+        carbonDataFiles.length,
+        ColumnarFormatVersion.V3,
+        null)
+      val dataFileFooter = converter.readDataFileFooter(blockInfo)
+      val blockletList = dataFileFooter.getBlockletList.asScala
+      for (blockletInfo <- blockletList) {
+        val dimensionColumnChunkReader =
+          CarbonDataReaderFactory
+            .getInstance
+            .getDimensionColumnChunkReader(ColumnarFormatVersion.V3,
+              blockletInfo,
+              carbonDataFiles.getAbsolutePath,
+              false).asInstanceOf[DimensionChunkReaderV3]
+        dimensionRawColumnChunks
+          .add(dimensionColumnChunkReader.readRawDimensionChunk(fileReader, blockIndex))
+      }
+      dimensionRawColumnChunks
+    }
+
+    def validateDictionary(rawColumnPage: DimensionRawColumnChunk,
```

Comment:
remove all dict related method, as we dont support it now

Reply:
done